### PR TITLE
[#2781] endless loop for channels.list 

### LINKED ISSUE #2781: endless loop for channels.list
## Describe the bug

- on `ui/channels` there is an endless loop of requests for` channels.list `

## To Reproduce

1- start a fresh instance from develop 
2- run the UI 
3- go to `ui/channels` and open the network tab

## Environment

- OS: iOS
- Browser: Chrome

### LINKED ISSUE #2784: path bug for facebook and instagram
## Describe the bug

The router paths are broken when navigating to create a new channel for `instagram` and` facebook`. 

The navigation should point to `/ui/channels/facebook/new` and `/ui/channels/instagram/new`

### PR DESCRIPTION
**Pull request by AudreyKj**
closes #2781 
closes #2784

**additional changes:**
disconnected channels errors fix
svg color fix

### CODE REVIEW

**Review by ljupcovangelski — APPROVED** [review]

File: frontend/ui/src/pages/Inbox/MessageInput/InputOptions.tsx [hunk 1]
```diff
@@ -64,7 +64,9 @@ export const InputOptions = (props: Props) => {
   useEffect(() => {
     if (!isFileLoaded) {
       setInputFile(null);
-      inputRef.current.value = null;
+      if (inputRef && inputRef.current) {
+        inputRef.current.value = null;
+      }
```
Comment (AudreyKj) [inline frontend/ui/src/pages/Inbox/MessageInput/InputOptions.tsx:69]: this was creating an error: when the component mounts, ref.current is not yet initialised

File: frontend/ui/src/pages/Inbox/MessageInput/index.tsx [hunk 2]
```diff
@@ -429,7 +429,7 @@ const MessageInput = (props: Props) => {
                 ref={textAreaRef}
                 rows={1}
                 name="inputBar"
-                placeholder={channelConnected && 'Enter a message...'}
+                placeholder={channelConnected ? 'Enter a message...' : ''}
                 autoFocus={channelConnected}
```
Comment (AudreyKj) [inline frontend/ui/src/pages/Inbox/MessageInput/index.tsx:433]: this was creating an error: when the channel was disconnected, the line was evaluated as false

File: lib/typescript/assets/scss/colors.scss [hunk 3]
```diff
@@ -2,6 +2,7 @@
   --color-text-contrast: #212428;
   --color-text-gray: #737373;
   --color-dark-elements-gray: #98a4ab;
+  --color-icons-gray: #a0abb2;
   --color-light-gray: #cad5db;
```
Comment (AudreyKj) [inline lib/typescript/assets/scss/colors.scss:6]: created a dedicated gray color for the icons

File: lib/typescript/assets/images/icons/paperclipChatplugin.svg [hunk 4]
```diff
@@ -1,43 +1 @@
...
-<g>
-</g>
-<g>
-</g>
-<g>
-</g>
-<g>
-</g>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 365.467 365.467" xml:space="preserve"><path d="M314.85 123.681 198.31 325.534c-22.027 38.153-70.988 51.272-109.141 29.244-38.152-22.027-51.271-70.987-29.244-109.14L182.935 32.58c17.972-31.128 57.917-41.832 89.046-23.86 31.128 17.972 41.832 57.918 23.86 89.046L192.147 277.37c-14.02 24.283-45.181 32.634-69.464 18.614s-32.632-45.182-18.612-69.465l97.225-168.398c2.841-4.922 9.135-6.608 14.056-3.767 4.922 2.841 6.608 9.135 3.767 14.057l-97.225 168.398c-8.346 14.456-3.376 33.006 11.079 41.352 14.456 8.346 33.006 3.375 41.352-11.081L278.019 87.476c12.298-21.301 4.974-48.635-16.327-60.934-21.301-12.298-48.636-4.974-60.934 16.327L77.748 255.928c-16.354 28.325-6.614 64.673 21.711 81.027s64.674 6.614 81.027-21.711l116.54-201.852c2.841-4.922 9.135-6.608 14.056-3.767 4.923 2.841 6.609 9.134 3.768 14.056z"/><g/><g/><g/><g/><g/><g/><g/><g/><g/><g/><g/><g/><g/><g/><g/></svg>
```
Comment (AudreyKj) [inline lib/typescript/assets/images/icons/paperclipChatplugin.svg:1]: optimized chat plugin template svg

**Review by thorstenairy — APPROVED** [review]
LGTM

**Pull request merged**